pkg/profiler/cpu/maps.go

```diff
@@ -39,8 +39,8 @@ const (
 
 	// With the current row structure, the max items we can store is 262k per map.
 	unwindTableMaxEntries = 100
-	maxUnwindTableSize    = 250 * 1000 // Always needs to be sync with MAX_UNWIND_TABLE_SIZE in BPF program.
-	unwindTableShardCount = 3
+	maxUnwindTableSize    = 250 * 1000 // Always needs to be sync with MAX_UNWIND_TABLE_SIZE in the BPF program.
+	unwindTableShardCount = 6          // Always needs to be sync with MAX_SHARDS in the BPF program.
```

Comment:
Not a blocker for this PR, but how about moving everything that needs to be synced to user space as you did in the previous PR?
We can use config and constants for these types of variables. We directly manipulate the map sizes before we run the program.
WDYT?

Reply:
You read my mind! 😄 . Was planning to do this for as many things as we can. Mind if we do it in another PR? Can open an issue if you are ok with this so we don't forget about this

Reply:
Yes, definitely we should do this in a separate PR and open it against the `main` branch for the next release. Feel free to open an issue for it.

Reply:
Sounds good! Will merge this then :)